parser/src/keys/mod.rs

```diff
@@ -374,6 +374,9 @@ pub fn str_to_oscode(s: &str) -> Option<OsCode> {
         "nop8" => OsCode::KEY_684,
         "nop9" => OsCode::KEY_685,
 
+        "mvmt" | "mousemovement" | "🖰mv" => OsCode::KEY_766,
```

Comment:
Needs a comment stating that this has no output mapping; this is only intended to be used in the input position alongside:

```
  mouse-movement-key mvmt
```

This should even be enforced within `parse_action_atom` of `parser/src/mod.rs` such that if found as an action, an error is generated.

Also, add a comment down at line 1163 that the key is being used as a dummy input for the mouse activity.